_projects/2017/sunpy/roi.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Region of Interest
+desc: Meta-Object creation for holding temporal and spatial features.
+# add a description of your project
+requirements:
+ - Familiarity with Python
+ - Understanding of design patterns
+difficulty: High
+issues:
+ - https://github.com/sunpy/sunpy/issues/164
+mentors: 
+ - dpshelio
+ - wafels
+initiatives:
+ - GSOC
+tags:
+ - sunpy
+collaborating_projects:
+ - sunpy
+---
+
+#### Description
+
+This project aims to plan and implement a object that could be used to
+define a region of interest and interact with the rest of data types used
+in sunpy.
+
+For example, a region of interest could be a particular are in the sun 
+on a particular moment in time. Therefore, that object should contain the limits
+of such area, its contour and the time when it was observed.
+It also needs other information, such as the coordinate system that it
+uses (notice that chaincode needs of a image pixel size too) and the wavelength
```

Comment:
the section in brackets is meaningless to someone who doesn't know what a chaincode is, I would drop it.